umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1351,6 +1351,122 @@
                                                                           :Unit "dB"
                                                                           :Value "10"}]
                                                        :Technique "Drunken Fist"}]}]}]})))))
+  (testing "GeographicCoordinateUnits migration from version 1.9 to 1.10"
+  (is (= {:HorizontalCoordinateSystem
```

Comment:
Looks like indentation is off here. The `(is ...)` blocks should be indented under the `(testing )` block

Reply:
Done.